docker-compose.yml

```
@@ -31,7 +31,7 @@ x-pg-exporter-env: &pg_exp_env
 # Services definitions
 services:
   nwaku:
-    image: ${NWAKU_IMAGE:-wakuorg/nwaku:v0.21.3}
+    image: ${NWAKU_IMAGE:-wakuorg/nwaku:v0.22.0-rc.0}
```

@vpavlin said `NWAKU_IMAGE=docker.io/wakuorg/nwaku:v0.22.0-rc.0` does it matter?

```suggestion
    image: ${NWAKU_IMAGE=docker.io/wakuorg/nwaku:v0.22.0-rc.0}
```

---

It works for me, I think that `docker.io` gets automatically resolved, and the `:-` makes `wakuorg/nwaku:v0.22.0-rc.0` the default value of `NWAKU_IMAGE` but if another value is set as env variable then the env variable has precedence.

Anyway, don't think it matters much but lmk what you think

---

leave it doesn't matter I guess.